policyengine-client/src/countries/country.jsx

```diff
@@ -115,7 +115,7 @@ export default class Country {
     }
 
     useLocalServer = false;
-    usePolicyEngineOrgServer = false;
+    usePolicyEngineOrgServer = true;
```

Comment:
```suggestion
    usePolicyEngineOrgServer = false;
```

When eventually merged